typescript/test/testdata/keymanager/src/v1/key_management_service_client.ts.baseline

```diff
@@ -977,7 +977,7 @@ export class KeyManagementServiceClient {
     options.otherArgs.headers[
       'x-goog-request-params'
     ] = gax.routingHeader.fromParams({
-      'crypto_key.name': request.cryptoKey!.name || '',
+      'crypto_key_name': request.cryptoKey!.name || '',
     });
```

Comment:
found this in nodejs-kms: https://github.com/googleapis/nodejs-kms/blob/master/src/v1/key_management_service_client.js#L1754
Is there a specific reason to change the params name to snakeCase?

Reply:
Thank you for spotting this! It's not intended, I'll fix it.

Reply:
Thanks! LGTM after the fix.

Reply:
Fixed now!